src/main/java/org/folio/rest/impl/InvoicesImpl.java

```diff
@@ -23,30 +23,30 @@
   private static final Logger logger = LoggerFactory.getLogger(InvoicesImpl.class);
   private static final String NOT_SUPPORTED = "Not supported";	// To overcome sonarcloud warning
   private static final String INVOICE_LOCATION_PREFIX = "/invoice/invoices/%s";
+  private static final String INVOICE_LINE_LOCATION_PREFIX = "/invoice/invoice-lines/%s";
+
 
   @Validate
   @Override
   public void postInvoiceInvoices(String lang, Invoice invoice, Map<String, String> okapiHeaders,
                                   Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-
-		InvoiceHelper helper = new InvoiceHelper(okapiHeaders, vertxContext, lang);
-
-		helper.createInvoice(invoice)
+    InvoiceHelper helper = new InvoiceHelper(okapiHeaders, vertxContext, lang);
+    helper.createInvoice(invoice)
       .thenAccept(invoiceWithId -> {
         Response response = PostInvoiceInvoicesResponse.respond201WithApplicationJson(invoiceWithId,
-          PostInvoiceInvoicesResponse.headersFor201()
-            .withLocation(String.format(INVOICE_LOCATION_PREFIX, invoiceWithId.getId())));
-        asyncResultHandler.handle(succeededFuture(response));
+        PostInvoiceInvoicesResponse.headersFor201()
+          .withLocation(String.format(INVOICE_LOCATION_PREFIX, invoiceWithId.getId())));
+          asyncResultHandler.handle(succeededFuture(response));
       })
       .exceptionally(t -> handleErrorResponse(asyncResultHandler, helper, t));
-	}
+  }
 
   @Validate
   @Override
   public void getInvoiceInvoices(int offset, int limit, String query, String lang, Map<String, String> okapiHeaders,
       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-  InvoiceHelper helper = new InvoiceHelper(okapiHeaders, vertxContext, lang);
-  helper
+    InvoiceHelper helper = new InvoiceHelper(okapiHeaders, vertxContext, lang);
+    helper
```

Comment:
why indent these two lines?  I think it was correct before

Reply:
I undid it.